packages/tocco-ui/src/DatePicker/useDatePickr.js

```diff
@@ -0,0 +1,61 @@
+import {useEffect, useState} from 'react'
+import '!style-loader!css-loader!flatpickr/dist/themes/light.css'
+import flatpickr from 'flatpickr'
```

Comment:
apparently, there's a plugin for a month picker called `monthSelectPlugin` which could be useful for the monthly mode (https://github.com/flatpickr/flatpickr/issues/269)

but I reckon you talked about this and decided that it wouldn't be worth the effort to implement a different picker for the monthly view.

Reply:
Haven't seen this plugin. Thanks. I talked to juan and we both think (at the moment) it's not worth the effort.